app/code/Magento/Multishipping/Test/Mftf/Test/StorefrontCreateOrderWithMultishippingAfterReturningToCartTest.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="StorefrontCreateOrderWithMultishippingAfterReturningToCartTest">
+        <annotations>
+            <features value="Multishipping"/>
+            <stories value="Checkout with multiple addresses."/>
+            <title value="Checkout with multiple addresses after returning on cart page during checkout."/>
+            <description value="Verify customer able to checkout with multiple addresses after returning to cart page and continue checkout with browser 'back' button."/>
+            <severity value="AVERAGE"/>
+            <testCaseId value="MC-39583"/>
```

Comment:
Please add MC-39583 as **has tests** link to task
Please specify test Path in MC-39583 zephyr
Please add **Expected result** for steps in MC-39583 zephyr